refresh/templates/services/pushnotifications/initializeBluemixService.swift

```diff
@@ -0,0 +1,6 @@
+
+    let pushNotificationsService = try manager.getPushSDKService(name: "<%- service.name %>")
+
+    let region = "<%- service.region %>".isEmpty ? PushNotifications.Region.US_SOUTH : "<%- service.region %>"
+
+    pushNotifications = PushNotifications(service: pushNotificationsService, region: region)
```

Comment:
I think you should pass the `region` straight in if you make the change to the prompt i.e. `region: "<%- service.region %>"`